platforms/r3-corda/configuration/samples/network-minikube.yaml

```diff
@@ -67,13 +67,13 @@ network:
       # Git Repo details which will be used by GitOps/Flux.
       # Do not check-in git_password
       gitops:
-        git_ssh: "gitops_ssh_url"         # Gitops ssh url for flux value files like "ssh://git@github.com/hyperledger-labs/blockchain-automation-framework.git"
+        git_ssh: "git@github.com:git_username/blockchain-automation-framework.git"         # Gitops ssh url for flux value files 
```

Comment:
This needs to be "ssh://git@github.com/<github_username>/blockchain-automation-framework.git" For all, also update same in all the sample network.yamls.